feat(#1670): rename and reorganize the design system section (#1701)
medic/cht-docs: content/en/design/user-experience-research/focus-group-discussions.md

---
title: "Focus Group Discussions"
linkTitle: "Focus Group Discussions"
weight: 2
description: >

>   Focus group discussions are a quick and convenient way to collect data from several users simultaneously. Use focus group discussions to gain insight into what users think about a topic or shared experience in a group setting and to learn about opinions, attitudes and preferences. 

Reviewer: ```suggestion
  How to collect data from several users simultaneously
```

---

> 

Reviewer: ```suggestion
Focus group discussions are a quick and convenient way to collect data from several users simultaneously. Use focus group discussions to gain insight into what users think about a topic or shared experience in a group setting and to learn about opinions, attitudes and preferences. 
```
